src/eval-frame/components/reps/tiny-rep.jsx

```diff
@@ -164,19 +164,19 @@ const typeToTinyRepMapping = {
 
 const handledTypes = Object.keys(typeToTinyRepMapping);
 
-export default class TinyRep extends React.Component {
+export default class TinyRep extends React.PureComponent {
   static propTypes = {
-    serializedObj: PropTypes.shape({
-      objClass: PropTypes.string.isRequired,
-      objType: PropTypes.string.isRequired,
-      size: PropTypes.number,
-      stringValue: PropTypes.string.isRequired,
-      isTruncated: PropTypes.bool.isRequired
-    })
+    objType: PropTypes.string.isRequired,
+    /* eslint-disable react/no-unused-prop-types */
+    isTruncated: PropTypes.bool.isRequired,
+    objClass: PropTypes.string.isRequired,
+    size: PropTypes.number,
+    stringValue: PropTypes.string.isRequired
```

Comment:
as above: spread props out individually rather than passing down a big object with children. facilitates shallow comparison of props.

(lesson: in general, we'll probably want to avoid PropTypes.shape in favor of flat props with simple types)